spec/models/miq_product_feature_spec.rb

```diff
@@ -16,16 +14,34 @@
     )
   end
 
+  it "is properly configured" do
+    everything = YAML.load_file(described_class.feature_yaml)
+    required_keys = [:identifier]
+    possible_keys = required_keys + [:name, :feature_type, :description, :children, :hidden, :protected]
```

Comment:
@jrafanie I updated in https://github.com/ManageIQ/manageiq/pull/9924/commits/75af1a2ff57193e54457d207a3bee2ada8b301d7 - does this take care of your concern? I thought about trying to infer the column names through some introspection but that looked way complicated